tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -79,7 +82,7 @@ sub run {
 
     # display test result
     # print the test output to the openQA output
-    my $cmd                       = "cd /tmp; zcat $upload_log_name.tar.gz | sed -n '/Executing VM installation/,/[0-9]* fail [0-9]* succeed/p'";
+    my $cmd = "cd /tmp; zgrep \"Overall guest upgrade result\" $upload_log_name.tar.gz; [ $? -eq 0 ] && zcat $upload_log_name.tar.gz | sed -n '/Overall guest upgrade result/,/[0-9]* fail [0-9]* succeed/p'";
```

Comment:
Do the check first seems to result in same output, no real benefit, doesn't it?

Reply:
Yes, that's true. I might not finish the two-part edits in one time, which cause the redundant checking. I am going to remove the condition statement.